www/js/main.js

```diff
@@ -275,31 +307,34 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
         });
         */
     }
-    $scope.editSyncConfig = function($event) {                  
-        $scope.settings.sync.new_config = JSON.parse(JSON.stringify($scope.settings.sync.config));              
-        console.log("settings popup = "+$scope.syncSettingsPopup);              
-        $scope.syncSettingsPopup.show($event);              
+
+    $scope.editSyncConfig = function($event) {
+        $scope.settings.sync.new_config = JSON.parse(JSON.stringify($scope.settings.sync.config));
+        console.log("settings popup = "+$scope.syncSettingsPopup);
+        $scope.syncSettingsPopup.show($event);
     }
-    $scope.saveAndReloadCollectSettingsPopover = function(result) {
+    
+    $scope.saveAndReloadSettingsPopup = function(result) {
```

Comment:
Hm, are we even using this? Or just the popover?
Did the pre-merge code actually have `saveAndReloadCollectSettingsPopover` AND `saveAndReloadSettingsPopover`?